repository-hbase/src/main/java/hbase/load/BulkLoader.java

```diff
@@ -48,17 +45,19 @@
     static final String REFERENCE_PERIOD = "hbase.load.period";
     static final String COLUMN_HEADINGS = "csv.column.headings";
     static final String ROWKEY_POSITION = "csv.id.position";
+    public static final String REVERSE_FLAG = "load.format.reverse";
     public static final String HEADER_STRING = "csv.header.string";
     private static final int SUCCESS = 0;
     private static final int ERROR = -1;
     private static final int MIN_ARGS = 5;
```

Comment:
min args is now 6

Reply:
ARG_HFILE_OUT_DIR and ARG_CSV_HEADER_STRING are optionally, so I think it should be 5.

Reply:
ok but if we have two optional params how do we know whether the sixth param passed is the out dir or the header string when only one optional param is supplied by the calling client?